helm/values.yaml

```diff
@@ -0,0 +1,75 @@
+# System
+replicaCount: 1
+
+nameOverride: ""
+fullnameOverride: ""
+
+images:
+  ucentralgwui:
+    repository: tip-tip-wlan-cloud-ucentral.jfrog.io/ucentralgw-ui
+    tag: master
```

Comment:
The branch is called `main` in this repository.

Reply:
Ah, thanks for noticing. Fixed